fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +193,44 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
```

Comment:
The return type is _not_ "perfectly reasonable jax-rs". Correct use of JAX-RS implies narrowly typed return values with corresponding mappers and serializers, not generically and opaquely typed return values. Correct usage here wold be one method with a return type of `RdfStream` for RDF and another with a return type of `StreamingOutput` for binaries, but we can't easily do that because of our funky URI structure.